tests/test_fuzzing.py

```diff
@@ -348,7 +347,6 @@ def test_fuzzing_incorrect(pcs_with_root):
         cli.fuzz_cmd,
         [
             "--cmd", "ls",
-            "--args", "-al",
             "--output-dir", ".",
         ],
     )  # fmt: skip
```

Comment:
Shouldn't the `--args` value be appended to the `--cmd`, or doesn't it matter for those tests?

Reply:
Good point, but it should not matter.